Makefile.toml

```diff
@@ -0,0 +1,101 @@
+#
+# Static checks
+#
+
+[tasks.format-check]
+workspace = false
+command = "cargo"
+args = ["fmt", "--", "--check"]
+
+[tasks.clippy]
+command = "cargo"
+args = ["clippy", "--all-targets", "--", "--deny=warnings"]
+
+[tasks.format-toml-check]
+description = "Check that all TOML files are formatted correctly"
+install_crate = { crate_name = "taplo-cli", version = "0.9.3", binary = "taplo", test_arg = "--help" }
+command = "taplo"
+args = ["format", "--check"]
+
+[tasks.doc-check]
+description = "Check documetation generation, similarly to how docs.rs will do it"
+command = "cargo"
+args = ["+nightly", "doc", "--all-features", "--no-deps"]
+env = { RUSTDOCFLAGS = "--cfg docsrs --deny=warnings" }
+install_crate = false
+
+#
+# Tests
+#
+
+[tasks.test-unit]
+command = "cargo"
+args = ["test", "--lib"]
+
+[tasks.test-integration]
+command = "cargo"
+args = ["test", "--no-run"]
```

Comment:
We don't want to run them?

Reply:
I want to be able to run `cargo make test-integration` from the root but only some projects define integration tests. What this actually does is run `cargo make test-integration` in each project and so each project may override this. I can see if I can slim this down any more so that it doesn't look like this gets run.